src/OrchardCore.Modules/OrchardCore.ContentTypes/ViewModels/EditPartViewModel.cs

```diff
@@ -32,13 +31,10 @@ public string DisplayName
 
         public string Description
         {
-            get { return Settings.Value<string>("Description"); }
-            set { Settings["Description"] = value; }
+            get { return PartDefinition.GetSettings<ContentPartSettings>().Description; }
+            set { PartDefinition.GetSettings<ContentPartSettings>().Description = value; }
         }
 
-        [BindNever]
-        public JObject Settings { get; set; }
-
         [BindNever]
```

Comment:
Never used, and code part's have no settings. Still need the empty constructor though.